modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1519,6 +1519,11 @@ CV_8U, CV_16U, or CV_32F, for larger aperture sizes, it can only be CV_8U.
  */
 CV_EXPORTS_W void medianBlur( InputArray src, OutputArray dst, int ksize );
 
+enum GaussianBlurFlags
```

Comment:
the flag to allow non-bit-exact implementations should be called universally and added universally to multiple functions. This very solution looks like quite ugly hack. Let's think of it and do properly. In IPP they used AlgorithmHint or something like that. Such enumeration as a separate proper enumeration could be added to various functions (starting with GuassianBlur, for example)